src/components/Gallery.js

```diff
@@ -1,22 +1,27 @@
-import React, { Component } from 'react';
-import Thumbnail from './Thumbnail.js';
-
+import React, { Component } from "react";
+import Thumbnail from "./Thumbnail";
+import GalleryImage from "./GalleryImage";
 // * CSS imports
-import '../css/Gallery.css';
+import "../css/Gallery.css";
 
 export default class Gallery extends Component {
   constructor(props) {
     super(props);
-    this.state = {
-
-    };
+    this.state = {};
+    this.shows = this.props.cardData.shows || [];
```

Comment:
Good point, that piece may be a holdover when I was trying to tests things out so if it moves out out or gets removed entirely is fine. I am curious about that constructor question though, i wouldn't assume it would call the constructor either.